src/status_im/contexts/settings/wallet/keypairs_and_accounts/actions/view.cljs

```diff
@@ -0,0 +1,7 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.actions.view
+  (:require [quo.core :as quo]))
+
+(defn view
+  [props]
+  [quo/drawer-top
+   (merge props {:blur? true})])
```

Comment:
Can be an `assoc`, It's makes more sense for a single kv and is faster.

Reply:
Yup yup, good point!
I think I'll try moving this to where I pass the props to avoid this extra `merge` 👍